SpatialGDK/Source/SpatialGDK/Public/Utils/ComponentReader.h

```diff
@@ -26,9 +26,38 @@ class ComponentReader
 							  USpatialActorChannel& Channel, bool& bOutReferencesChanged);
 
 private:
+	struct ApplySchemaObjectDataStruct
```

Comment:
When I broke out part of ApplySchemaObject into ApplySchemaObjectFields, I was cringing at how many parameters I had to pass in, so I tried to make it feel more sane by making this struct. I'm not sure that is the best solution, though, since the struct is only used in one spot -- thoughts?